src/tests/dhcpserver/DhcpServer_Robustness_Test_001.test.js

```diff
@@ -0,0 +1,48 @@
+import constants from '../../commonMethods/constants'
+import { pluginActivate, pluginDeactivate } from '../../commonMethods/controller'
+import { getCpuLoad } from '../../commonMethods/deviceInfo'
+
+export default {
+  title: 'DHCP Server plugin robustness test',
+  description:
+    'Activates and Deactivates DHCP Server plugin repeatedly and checks the functionality',
+  context: {
+    cpuLoad: 90,
+  },
+  steps: [
+    {
+      title: 'Activate and deactivate DHCP Server plugin for 30 times',
+      description: 'Nested test to repeat the test for 30 times',
+      repeat: 30,
+      steps: [
+        {
+          description: 'Deactivate DHCP Server plugin and check if it is deactivated',
+          test: pluginDeactivate,
+          params: constants.dhcpserver,
+          assert: 'deactivated',
+        },
+        {
+          description: 'Activate DHCP Server Plugin and check if it is activated',
+          test: pluginActivate,
+          params: constants.dhcpserver,
+          assert: 'activated',
+        },
```

Comment:
Same here, should we get the list of plugins and check the state?

Reply:
'pluginActivate' activates the plugin and also **returns the state of the plugin** and it is asserted **using the state as 'activated'**. I think no need of getting the list of active plugins from controller. Please let me know if we can still do as you mentioned.